packages/block-editor/src/components/media-replace-flow/index.js

```diff
@@ -171,21 +122,22 @@ const MediaReplaceFlow = ( {
 								} }
 							/>
 						</MediaUploadCheck>
-						{ onSelectURL && (
-							<MenuItem
-								icon={ link }
-								onClick={ () =>
-									setShowURLInput( ! showURLInput )
-								}
-								aria-expanded={ showURLInput }
-							>
-								<div> { __( 'Insert from URL' ) } </div>
-							</MenuItem>
-						) }
 					</NavigableMenu>
-					{ showURLInput && (
+					{ onSelectURL && (
 						<div className="block-editor-media-flow__url-input">
-							{ urlInputUIContent }
+							<span className="block-editor-media-replace-flow__image-url-label">
+								{ __( 'Current media URL:' ) }
+							</span>
+							<LinkControl
+								value={ { url: mediaURLValue } }
+								settings={ [] }
```

Comment:
Is this empty value required in order for `LinkControl` to function? If so then we need to raise an issue to make sure it isn't required.

Reply:
Yes it is required. There is a default setting of showing `openInNewTab` option and in case of editing URLs you don't need that.